Source/JavaScriptCore/disassembler/ARM64Disassembler.cpp

```diff
@@ -36,6 +36,9 @@ namespace JSC {
 
 bool tryToDisassemble(const CodePtr<DisassemblyPtrTag>& codePtr, size_t size, void* codeStart, void* codeEnd, const char* prefix, PrintStream& out)
 {
+    constexpr int commentAlignment = 64;
+    constexpr auto* commentFormatStart = "\033[1;30m";
+    constexpr auto* commentFormatEnd = "\033[0m";
```

Comment:
Please keep plain text. It is really hard for dumping it into a simple text format and viewing in the editor.

Reply:
Perhaps the middle ground is to use escape sequences when connected to tty, and plain text otherwise?